### ISSUE TITLE
OpenSSL warning due to passphrase caching

### ISSUE THREAD

**Issue by poelstra-sioux** [openxpki/clca#17]
When running `clca` commands, the following is printed:

```
Enter passphrase:
*** WARNING : deprecated key derivation used.
Using -iter or -pbkdf2 would be better.
```

It appears that the warning is emitted from the process of caching the passphrase.

One way to fix it, is to add `-pbkdf2` to the `openssl enc` command, as suggested by it.

On line 98:
```sh
        ENCRYPTED_PASSPHRASE=`echo "$CLEARTEXT_PASSPHRASE" | $OPENSSL enc -aes256 -pbkdf2 -a -e -pass env:CLCA_SESSION_KEY`
```

And line 110:
```sh
        PASSPHRASE=`echo "$ENCRYPTED_PASSPHRASE" | $OPENSSL enc -aes256 -pbkdf2 -a -d -pass env:CLCA_SESSION_KEY`
```

Thanks for this helpful tool!

**Comment by Sult** [comment]
You could set the `DEFAULT_ENC_ALGORITHM` parameter in `clca.cfg` to `pbkdf2`
This fixes the issue and also improves security a bit

**Comment by mbartosch** [comment]
fixed in f501d98

**Comment by mbartosch** [comment]
apologies - it's pushed now.
